getting_started/first_3d_game/09.adding_animations.rst

```diff
@@ -242,10 +254,10 @@ Animating the mobs
 Here's another nice trick with animations in Godot: as long as you use a similar
 node structure, you can copy them to different scenes.
 
-For example, both the *Mob* and the *Player* scenes have a *Pivot* and a
-*Character* node, so we can reuse animations between them.
+For example, both the ``Mob`` and the ``Player`` scenes have a ``Pivot`` and a
+``Character`` node, so we can reuse animations between them.
 
-Open the *Player* scene, select the animation player node and open the "float" animation.
+Open the ``Player.tscn`` scene, select the ``AnimationPlayer`` node and open the "float" animation.
 Next, click on **Animation > Copy**. Then open ``Mob.tscn`` and open its animation
```

Comment:
I wasn't able to find these buttons, however I was able to find similar commands in the Animation -> Manage Animations menu, though I'm not sure thats a good / recommended way to copy/paste animations.

Reply:
You are right. I am trying to find these buttons myself, and it seems I meant 'copy tracks' which is a horrible way to copy animations. Or I did the same thing as you with `ManageAnimations`
Anyway, I am a newbie so I do not know what is the recommended/ideal way to copypaste animations.